src/improvements/README.md

```diff
@@ -41,23 +41,23 @@ This toml configuration file allows task developers to set gas limits for the ta
 #### Template 00 to set gas configs:
 
 ```bash
-forge script test/task/mock/example/template/GasConfigTemplate.sol --sig "run(string)" test/task/mock/example/task-00/mainnetConfig.toml --rpc-url mainnet -vvv
+forge script src/improvements/template/GasConfigTemplate.sol --sig "run(string)" test/tasks/mock/example/task-00/config.toml --rpc-url mainnet -vvv
 ```
 
 #### Template 01 to set dispute game upgrade:
 
 ```bash
-forge script test/task/mock/example/template/DisputeGameUpgradeTemplate.sol --sig "run(string)" test/task/mock/example/task-01/mainnetConfig.toml --rpc-url mainnet -vvv
+forge script src/improvements/template/DisputeGameUpgradeTemplate.sol --sig "run(string)" test/tasks/mock/example/task-01/mainnetConfig.toml --rpc-url mainnet -vvv
```

Comment:
Here this is broken the config chain the name into the solidity file: 
Not related to your change seems so but we can at least maybe change it? 

![CleanShot 2025-02-05 at 17 03 01@2x](https://github.com/user-attachments/assets/731c73d7-87a3-424d-a469-40e677d61011)

Reply:
need to fix by: 
```
test/tasks/mock/example/task-01/config.toml
```suggestion
forge script src/improvements/template/DisputeGameUpgradeTemplate.sol --sig "run(string)" test/tasks/mock/example/task-01/config.toml --rpc-url mainnet -vvv
```
```